src/kafka/consumer.js

```diff
@@ -735,7 +735,7 @@ class Consumer extends EventEmitter {
    * @param {string} metadataOptions.topic - Topic string for which to fetch
    * metadata
    * @param {number} metadataOptions.timeout - Max time, in ms, to try to fetch
-   * metadata before timing out. Defaults to 3000.
+   * metadata before timing out. Defaults to 30,000 (30 seconds).
```

Comment:
Why has this been changed? where did you get this value from?

Reply:
This hasn't been changed, the documentation is wrong.

From the linked ticket: (https://github.com/mojaloop/project/issues/816)
>According to node-rdkafka client, this default is actually 30,000ms: https://github.com/Blizzard/node-rdkafka/blob/be47c8b5422fc39c5b41dab33850cd98d89f03b7/lib/client.js#L123

Reply:
This is the current master version https://github.com/Blizzard/node-rdkafka/blob/e4adf30e062e0efe1c7b07f3d63a0702d506a393/lib/client.js#L325

Is the link you shared from the release?

Reply:
From the underlying GetMetadata method in the C++, it looks like it defaults to 30000 ms:

https://github.com/Blizzard/node-rdkafka/blob/19c27ea9630b17759aad4141248b77547749bb90/src/connection.cc#L241